# HG changeset patch
# User Ian Campbell <ian.campbell@citrix.com>
# Date 1295284460 0
# Node ID 0dbad563a6592addd1d757afa41b3960d0862270
# Parent  f1a5ac39c15e30b1f5be190f27b4cf914b3d9723
tools/blktap, blktap2: include <sys/mount.h> instead of <linux/fs.h>

The former is a userspace sanitised header which contains the
definitions we need. In some distros linux/fs.h defines WRITE which
conflicts with blktaps own use of that name.

Also there is no reason to use <linux/errno.h> over the more normal
<errno.h>.

Signed-off-by: Ian Campbell <ian.campbell@citrix.com>
Committed-by: Ian Jackson <ian.jackson@eu.citrix.com>

diff -r f1a5ac39c15e -r 0dbad563a659 tools/blktap/drivers/blk_linux.c
--- a/tools/blktap/drivers/blk_linux.c	Thu Jan 13 16:00:59 2011 +0000
+++ b/tools/blktap/drivers/blk_linux.c	Mon Jan 17 17:14:20 2011 +0000
@@ -1,6 +1,6 @@
 #include <inttypes.h>
 #include <sys/ioctl.h>
-#include <linux/fs.h>
+#include <sys/mount.h>
 #include "tapdisk.h"
 #include "blk.h"
 
diff -r f1a5ac39c15e -r 0dbad563a659 tools/blktap2/drivers/blk_linux.c
--- a/tools/blktap2/drivers/blk_linux.c	Thu Jan 13 16:00:59 2011 +0000
+++ b/tools/blktap2/drivers/blk_linux.c	Mon Jan 17 17:14:20 2011 +0000
@@ -1,7 +1,7 @@
 #include <inttypes.h>
+#include <errno.h>
 #include <sys/ioctl.h>
-#include <linux/fs.h>
-#include <linux/errno.h>
+#include <sys/mount.h>
 #include "tapdisk.h"
 #include "blk.h"
 
